### ISSUE TITLE
Redo the roads, and examine/redo building scale in the city.

### ISSUE THREAD

**Issue by CurryKitten** [CurryKitten/CurryKitten-Sim#42]
A sim user kindly took the time to create a video showing some weird gaps he found in the road joins and som grass that was somehow growing up through the road :)  https://youtu.be/NZFNyNAN5oM

The grass is easy to fix, and the gaps should be reasonably so - but in general, the road needs redoing - it would be a deathtrap for any vehicle going down there.  I also need to think about how I might link the city up with a bridge that people keep asking for.

**Comment by CurryKitten** [comment]
I've fixed the grass growing through the roads, but I'm not going to touch the gaps yet until I redo the road completely - at which point I'll also look at the scale of the buildings to make sure they look correct in comparison to a quad - the feeling is they may all be a bit small.

**Comment by CurryKitten** [comment]
I took a look at the buildings today and measured a floor by stacking up 3 1m cubed blocks

![Screenshot 2020-04-20 at 16 02 55](https://user-images.githubusercontent.com/12443664/79768678-03db0200-8323-11ea-8bdd-1b6bf8b5bb8d.jpg)

This shows each floor (in this building at least) having a height of around 2.5m.  Checking with a few colleagues, the accepted "norm" in commercial buildings is around 3m per floor, so expanding this out in all directions until we just about lined up with 3 blocks

![Screenshot 2020-04-20 at 16 20 52](https://user-images.githubusercontent.com/12443664/79768812-3258dd00-8323-11ea-8376-8f12a4676e08.jpg)

I'm not such the scale factor will be consistent in all the buildings, I need to go around and check them all out, and then ensure that the colliders also scale up correctly.

**Comment by CurryKitten** [comment]
I've redone the scale of all the buildings - taking a bit more artistic license with some of them to make them fit nicer in the city - this has also created a few nice tight gaps.

I had to make a few subtle changes to the plane waypoints to stop them hitting the newly grown buildings.

I also went back and excavated a load of sand in order to lower the roads, so the road is now completely flat.

![Screenshot 2020-04-21 at 15 31 11](https://user-images.githubusercontent.com/12443664/79878751-b4113f00-83e5-11ea-9cf3-5106bb75db0c.jpg)
